.travis.yml

```diff
@@ -1,3 +1,11 @@
+# Since we use an API key for Travis CI we need to use the https url, and since
+# we use the ssh url privately we need to handle git submodules ourselves
+git:
+  submodules: false
+# Use sed to replace the SSH URL with the public URL, then initialize submodules
+before_install:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
```

Comment:
I think there is a [git command to do this](http://stackoverflow.com/a/928539)

Something like: `git config submodule.fonttools.url https://github.com/behdad/fonttools.git`